com.vrcfury.vrcfury/Editor/VF/Service/OnDemandSyncService.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using VF.Builder;
+using VF.Feature.Base;
+using VF.Injector;
+using VF.Utils.Controller;
+using VRC.SDK3.Avatars.Components;
+
+namespace VF.Service {
+    [VFService]
+    public class OnDemandSyncService {
+        [VFAutowired] private readonly AvatarManager manager;
+        [VFAutowired] private readonly MathService math;
+        [VFAutowired] private readonly DirectBlendTreeService directTree;
+        [VFAutowired] private readonly SmoothingService smoothingService;
+        
+        private readonly List<VFAFloat> onDemandSyncParameters = new List<VFAFloat>();
```

Comment:
This should probably be a HashSet<string>, and then the type re-verified and VFAFloat aquired later on. I can already see this having duplicates.